src/main/java/org/spongepowered/common/mixin/invalid/api/mcp/block/state/StateImplementationMixin_API.java

```diff
@@ -68,7 +68,7 @@
 import javax.annotation.Nullable;
 
 /**
- * This shares implementation with {@link BlockStateMixin_API}, since this
+ * This shares implementation with {@link BlockStateMixin_API_Old}, since this
```

Comment:
It shouldn't be.

Reply:
I'd assume the old mixin shouldn't have a shared impl, but the way you've put it makes me think this mixin doesn't really belong here.